src/pages/components/tipSection/index.js

```diff
@@ -1,11 +1,40 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import Tips from './Tips';
 import IdeaIcon from '../icons/IdeaIcon';
+import Demo from './Demo';
 
 const TipSection = () => {
   const { t } = useTranslation('features');
 
+  const [isDemo, setIsDemo] = useState(false);
+  const [isActive, setIsActive] = useState(true);
+  const [isNotActive, setIsNotActive] = useState(true);
+
+  useEffect(() => {
+    if (isDemo === false) {
```

Comment:
```suggestion
    if (!isDemo) {
```

@CaneteJoash you can use this instead.